src/pages/EndpointManager.vue

```diff
@@ -103,12 +103,11 @@ export default {
     },
     uiThemeCategory: {
       get() {
-        let zclProps = this.$store.state.zap.selectedZapConfig?.zclProperties
-        // Picking the first category in the case of multi-protocol(zigbee/matter)
-        if (Array.isArray(zclProps) && zclProps.length > 0) {
-          return zclProps[0].category
+        if (this.$store.state.zap.isMultiConfig) {
+          return 'multiprotocol'
         } else {
-          return this.$store.state.zap.selectedZapConfig?.zclProperties.category
+          return this.$store.state.zap.selectedZapConfig?.zclProperties[0]
```

Comment:
Same as above. Is `zclProperties` an array of objects or just an object based on multi-protocol or single protocol?